encodedBicoding/Banka-react
# ft/168053451 Implements user dashboard, UI and functionality

#### What does this PR do?
>- implements uI and functionality for user dashboard
#### Description of Task to be completed?
>- create components and consume endpoint
#### How should this be manually tested?
>- clone the repo
>- run npm install - to install all dependencies
>- run npm start dev
>- create an account using the provided UI, you should see a dashboard
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[ft/168053451]()
#### Screenshots (if appropriate)
N/A
#### Questions:
